pkg/tsdb/loki/instrumentation/instrumentation.go

```diff
@@ -0,0 +1,34 @@
+package instrumentation
+
+import (
+	"context"
+	"time"
+
+	"github.com/grafana/grafana/pkg/infra/tracing"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	pluginParsingResponseDurationSeconds = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
FYI @marefr @andresmgot the only other instance of `promauto` being used in core plugins is here:

https://github.com/grafana/grafana/blob/bb7acf9e47e82e5a78d6addfc6ce78b478078199/pkg/tsdb/prometheus/instrumentation/instrumentation.go#L10

I think this will be a problem for the decoupling core plugins work. We have `CollectMetrics` interface for non-core plugins but I think we can't query that data in HG yet.

Do you think this will be a problem?

Reply:
Just to provide context, this is already used in (for example) Prometheus data source https://github.com/grafana/grafana/blob/main/pkg/tsdb/prometheus/instrumentation/instrumentation.go.

Reply:
I don't see any problem with promauto per se as of now since the plugin SDK is using the prometheus default gatherer. The potential problem might be the namespace, but as long as that is not changing should be fine. I think its fine for now.

Reply:
Thank you! :pray: